conf/npm/prepublishOnly.sh

```diff
@@ -4,3 +4,4 @@
 
 mv ./README.md ./README.tmp
 cp ./conf/npm/README.md ./README.md
+npm run generate-entry-points
```

Comment:
if we're running this as part of the prepublish, should we still check the exports into the package.json?

Reply:
I think keeping it always checked in is simpler. Maybe it would make sense to run the generation as part of the build script though?

Reply:
I was a little worried about running generate-entry-points blindly as part of the prepublish, since npm doesn't let you undo a publish. On the other hand we can just publish a patch version if something goes wrong, though it's a little annoying.

Reply:
oh yeah, doing in build step would be better. Will update!